src/app/pages/ArticlePage/index.stories.tsx

```diff
@@ -230,3 +236,25 @@ export const ArticlePageWithElectionBanner = {
     chromatic: { disableSnapshot: true },
   },
 };
+
+export const ArticlePageWithLiteSiteCTA = (
+  _: StoryArgs,
+  { service }: StoryProps,
+) => (
+  <ComponentWithContext
+    data={articleData}
+    service={service}
+    liteSiteCTAEnabled
+  />
+);
+
+export const TestArticlePageWithLiteSiteCTA = {
```

Comment:
One last one - should we have a second snapshot test for RTL with arabic article data?